test/onnx/test_onnx_export.py

```diff
@@ -29,9 +30,9 @@ def export_to_onnx(
             Union[contextlib.AbstractContextManager, contextlib.ContextDecorator],
         ]
     ] = None,
-    mocks: Optional[Iterable[unittest.mock.patch]] = None,
```

Comment:
`unittest.mock.patch` is not a correct type. since the types are hidden unittest.mock, I removed the types for now.